google/google_timesheets.py

```diff
@@ -228,8 +244,22 @@ def get_consolidated_df_for_week(items: list, users: list, year: int, week: str,
 
             # drop any zero total rows
             df_timesheet = df_timesheet.loc[df_timesheet["total"]>0,:]
-            df = pandas.concat([df,df_timesheet])
-            print(df)
+            print(df_timesheet)
+
+            # Consolidate duplicate rows by summing them
+            df_timesheet = df_timesheet.groupby(["client_code", "task_code", "name"]).agg({day: 'sum' for day in DAYS})
```

Comment:
Added an aggregator while consolidating the same client code and task code instead of throwing duplicate key entries exception. My week 37 is a good example of consolidation.

Reply:
Cool - checking it out - thank you for pointing out test data.